components/common/ContentBox.tsx

```diff
@@ -1,19 +1,22 @@
+import { ReactElement, ReactNode } from 'react'
 import { useTheme, Box } from '@chakra-ui/react'
-
-export const RESPONSIVE_PADDING = [4, null, 8]
+import { RESPONSIVE_PADDING } from '../lib/Definitions'
```

Comment:
This first comment isn't TypeScript specific, but...I much prefer keeping constants in modules where they are relevant, rather than off somewhere in a big long list of everything.  I think those long lists are harder to understand and harder to maintain. And I think `RESPONSIVE_PADDING` is very strongly related to the module where I defined it. It's not just any responsive padding, it's the responsive padding for the page-scale content that `ContentBox` is a wrapper for. While you can use it without using a `ContentBox`, I'd expect to see it being imported from there to even know what it means.

Maybe this wasn't super clear because it wasn't really clear what `ContentBox` was intended to be? I definitely struggled with the name. The idea is that it adds consistent max page width plus padding (and that padding gets bigger as you go from phones to larger displays). But you don't wrap the whole page in it (since you may have full-bleed elements that span beyond it), but rather you wrap particular content that you want to constrain to the comfortable page width.

Reply:
Types should be local to where they are used, until they become so widely used that we need to centralize them.

Reply:
Yeah, I'll revert this, initially I thought it was going to be needed elsewhere, but by the end that wasn't true! Great catch and reminder about keeping them local!